solanches/models.py

```diff
@@ -79,3 +79,9 @@ def get_all():
     def to_dict(self):
         comercio = vars(self).copy()
         return comercio
+    
+    @staticmethod
+    def get_by_name(name):
+        query = {"name": name}
```

Comment:
O nome do atributo no objeto é `nome`
```suggestion
        query = {"nome": name}
```